src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private int index = new Random().nextInt(Ball.Color.values().length);
```

Comment:
you have to generate a random num each time you call method getRandomColor, now you generating while ColorSupplier initialization, so the number is the same for each class obj